packages/consumption/src/modules/notifications/itemProcessors/attributeDeleted/PeerSharedAttributeDeletedByPeerNotificationItemProcessor.ts

```diff
@@ -0,0 +1,80 @@
+import { ILogger } from "@js-soft/logging-abstractions";
+import { PeerSharedAttributeDeletedByPeerNotificationItem } from "@nmshd/content";
+import { CoreDate, TransportLoggerFactory } from "@nmshd/transport";
+import { ConsumptionController } from "../../../../consumption/ConsumptionController";
+import { CoreErrors } from "../../../../consumption/CoreErrors";
+import { PeerSharedAttributeDeletedByPeerEvent } from "../../../attributes";
+import { DeletionStatus, LocalAttributeDeletionInfo } from "../../../attributes/local/LocalAttributeDeletionInfo";
+import { ValidationResult } from "../../../common";
+import { LocalNotification } from "../../local/LocalNotification";
+import { AbstractNotificationItemProcessor } from "../AbstractNotificationItemProcessor";
+
+export class PeerSharedAttributeDeletedByPeerNotificationItemProcessor extends AbstractNotificationItemProcessor<PeerSharedAttributeDeletedByPeerNotificationItem> {
+    private readonly _logger: ILogger;
+
+    public constructor(consumptionController: ConsumptionController) {
+        super(consumptionController);
+        this._logger = TransportLoggerFactory.getLogger(PeerSharedAttributeDeletedByPeerNotificationItemProcessor);
+    }
+
+    public override async checkPrerequisitesOfIncomingNotificationItem(
+        notificationItem: PeerSharedAttributeDeletedByPeerNotificationItem,
+        notification: LocalNotification
+    ): Promise<ValidationResult> {
+        const attribute = await this.consumptionController.attributes.getLocalAttribute(notificationItem.attributeId);
+
+        if (typeof attribute === "undefined") {
+            return ValidationResult.success();
```

Comment:
As I understood it, we don't really use logs, since nobody looks at them anyways. If we want to check if an Attribute of a given Attribute ID was deleted by the peer, we will have the LocalNotification to look it up. @jkoenig134 What is your opinion on this?